CDS-28 Create boilerplate for non-landing pages

Liannus opened the pull request:
Changelog:
- Added boilerplate for formatting of non-landing pages

Liannus committed:
Format non-landing pages

forkball commented on src/components/ContactLinks/ContactLinks.scss:
```diff
@@ -0,0 +1,38 @@
+  .text {
+    font-size: 1.5em;
+    font-weight: 300;
+    word-break: break-word;
```
We should probably use tailwind for word-break

Liannus replied:
Resolved

forkball commented on src/components/ContactLinks/ContactLinks.scss:
```diff
@@ -0,0 +1,38 @@
+    font-size: 1.5em;
+    margin-right: 0.2em;
+    font-weight: 300;
+    padding: 0;
```
Padding should be done with tailwind

Liannus replied:
resolved

forkball commented on src/components/ContactLinks/ContactLinks.scss:
```diff
@@ -0,0 +1,38 @@
+  }
+
+  .linebreak {
+    height: 3px;
```
Should be down with tailwind

Liannus replied:
Tailwind doesn't go small enough, `h-1`: 0.25rem looks odd imo.

forkball replied:
Hmm fair enough then.

forkball commented on src/components/ContactLinks/ContactLinks.scss:
```diff
@@ -0,0 +1,38 @@
+
+  .icon {
+    font-size: 3em;
+    margin: 0.2em 0.5em;
```
Should be done with tailwind

Liannus replied:
Resolved

forkball reviewed (commented):
Some CSS stylings should be done with tailwind, otherwise looks good.

Liannus committed:
updates for pr

forkball commented on src/components/Card/Card.scss:
```diff
@@ -0,0 +1,14 @@
+  color: $color-off-white;
+  font-family: "Open sans";
+
+  position: relative;
```
Should be done with tailwind, missed this on my first look.

Liannus replied:
resolved

Liannus replied:
my b, resolved

forkball commented on src/components/Card/Card.scss:
```diff
@@ -0,0 +1,14 @@
+
+  .inner-container {
+    background-color: $color-dark-grey;
+    margin: -0.75rem;
```
Also should be done with tailwind.

Liannus replied:
can't do negative margins with tailwind.

Liannus replied:
my b, resolved

forkball commented on src/components/Card/Card.scss:
```diff
@@ -0,0 +1,14 @@
+  font-family: "Open sans";
+
+  position: relative;
+  z-index: 2;
```
Can be done in tailwind, but should it be?

Liannus replied:
yep, resolved

forkball commented on src/components/ContactLinks/ContactLinks.scss:
```diff
@@ -0,0 +1,35 @@
+
+  .textIcon {
+    font-size: 1.5em;
+    margin-right: 0.2em;
```
Should be done in tailwind, missed this on first look.

Liannus replied:
resolved

forkball reviewed (changes requested):
Sorry, missed some stuff on my first look.

Liannus committed:
update dynamic fonts and fix commits for pr

Liannus commented:
Also changed dynamic-fonts to `dynamic-header` and `dynamic-text` with slightly different breakpoints that I think work better and updated some fonts on cards and headers that didn't have them already.
